crates/language/src/buffer.rs

```diff
@@ -3148,7 +3148,7 @@ fn indent_size_for_line(text: &text::BufferSnapshot, row: u32) -> IndentSize {
     indent_size_for_text(text.chars_at(Point::new(row, 0)))
 }
 
-fn indent_size_for_text(text: impl Iterator<Item = char>) -> IndentSize {
+pub fn indent_size_for_text(text: impl Iterator<Item = char>) -> IndentSize {
```

Comment:
I exported and used this function directly but I could also add an `impl` function for `Inlay`. Though I don't think it's worth the indirection for a singular use case.